### PR TITLE
Fixed testing, implemented validateLab

### PR DESCRIPTION
**Pull request by cemersoz**

### CODE REVIEW

File: collections/tuxlab.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+var tuxlab = function(){}
```
Comment (DerekTBrown) [inline collections/tuxlab.js]: This shouldn't be in collections.  This should be in the /lab folder

File: tests/example_data/labfile1.js [hunk 2]
```diff
@@ -38,3 +38,4 @@ tuxlab.tasks = function(env){
            .nextTask(task4);
 }
 module.exports = tuxlab;
+//export tuxlab
```
Comment (DerekTBrown) [inline tests/example_data/labfile1.js]: Comments above code

File: collections/tuxlab.js (hunk 1 above)
Comment (cemersoz) [inline collections/tuxlab.js]: labfiles reference the tuxlab module. we can either publish it to npm or force them to use a relative path. I figured forcing a ./tuxlab.js path would be easiest..
or, if you'd prefer I can manipulate lab strings to reference the correct file. i.e., instructors won't add import and export I'll just add it to the beginning and end of the string. Your call

File: tests/example_data/labfile1.js (hunk 2 above)
Comment (cemersoz) [inline tests/example_data/labfile1.js]: fixed this

**Pull request merged**